src/components/Admin/AIAnalyticsSummarySkeleton.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Skeleton, Stack } from '@edx/paragon';
+
+const AIAnalyticsSummarySkeleton = () => (
+  <Stack direction="horizontal" gap={2}>
+    <Skeleton height={40} width={250} />
+    <Skeleton height={40} width={250} />
+  </Stack>
```

Comment:
instead of the custom skeleton, can we use a built-in skeleton like [this](https://github.com/openedx/frontend-app-admin-portal/blob/6a47c984164d0bf7c2448daaa089ac6908eb7fcd/src/components/learner-credit-management/BudgetCard-V2.jsx#L33) in the original component?

Reply:
@jajjibhai008 as far as I know the built in skeleton is for cards, here we want to show a skeleton for buttons. Please let me know if I have missed something.